src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
```

Comment:
```suggestion
    this.hidden = !this.hidden;
```
![image](https://user-images.githubusercontent.com/38065883/152378344-f7ca3515-9b3c-403b-b608-391abdee7d52.png)
changes to the opposite